microcosm_postgres/factories/sessionmaker.py

```diff
@@ -1,9 +1,47 @@
+from microcosm.api import defaults
+from microcosm.config.types import boolean
+from microcosm.config.validation import typed
+from microcosm.scoping.factories import ScopedFactory
+from microcosm.scoping.proxies import ScopedProxy
+from sqlalchemy.orm import Session
 from sqlalchemy.orm import sessionmaker
 
 
+ENGINE = "__engine__"
+
+
+def unwrap(bind):
+    return bind.__component__ if isinstance(bind, ScopedProxy) else bind
+
+
+@defaults(
+    # enable multiple engines (binds) via ScopedFactory
+    multi=typed(boolean, default_value=True)
```

Comment:
Defaults to multi?

Reply:
Good catch. Left over from local testing.